sync-todo/v2/client/swiftui/App/AppConfig.swift

```diff
@@ -8,12 +8,12 @@ struct AppConfig {
     var baseUrl: String
 }
 
-/// Read the Realm.plist file and store the app ID and baseUrl to use elsewhere.
+/// Read the atlasConfig.plist file and store the app ID and baseUrl to use elsewhere.
 func loadAppConfig() -> AppConfig {
     guard let path = Bundle.main.path(forResource: "Realm", ofType: "plist") else {
-        fatalError("Could not load Realm.plist file!")
+        fatalError("Could not load atlasConfig.plist file!")
     }
-    // Any errors here indicate that the Realm.plist file has not been formatted properly.
+    // Any errors here indicate that the atlasConfig.plist file has not been formatted properly.
     // Expected key/values:
     //      "appId": "your Realm app ID"
```

Comment:
```suggestion
    //      "appId": "your App Services app ID"
```

Reply:
I can't comment directly on the lines below, but we should probably rename `realmPropertyList` in ln 20 below to `atlasConfigPropertyList` or something along those lines to go along with the renaming. And then update ln 21 & ln 22 to get the appId and baseUrl from it the renamed var.